### PR TITLE
Add workaround for ssh tunnel connections to Rebex SSH server 

### PR DESCRIPTION
**Pull request by stmaute**
Add  possibility to activate the workaround by setting an entry in the specificConfig;

### CODE REVIEW

File: connectivity/service/src/main/java/org/eclipse/ditto/connectivity/service/messaging/tunnel/SshTunnelActor.java [hunk 1]
```diff
@@ -177,7 +177,8 @@ private void handleConnectResult(final ConnectFuture connectFuture) throws IOExc
             logger.debug("SSH session connected successfully.");
             sshSession = connectFuture.getClientSession();
             sshSession.addSessionListener(new TunnelSessionListener(getSelf(), logger));
-            sshSession.addChannelListener(new TunnelChannelListener(getSelf(), logger));
+            sshSession.addChannelListener(new TunnelChannelListener(getSelf(),
+                    this.connection.getSpecificConfig().getOrDefault("initialWindowSize","0"), logger));
```
Comment (dguggemos) [inline connectivity/service/src/main/java/org/eclipse/ditto/connectivity/service/messaging/tunnel/SshTunnelActor.java:181]: Maybe we should call it e.g. "initialSshChannelWindowSize" to be more precise what is affected by the config property.

File: connectivity/service/src/main/java/org/eclipse/ditto/connectivity/service/messaging/tunnel/TunnelChannelListener.java [hunk 2]
```diff
@@ -46,14 +57,22 @@ public void channelInitialized(final Channel channel) {
 
     @Override
     public void channelOpenSuccess(final Channel channel) {
+        if (initialWindowSize > 0) {
+            // workaround for Rebex SSH server
```
Comment (dguggemos) [inline connectivity/service/src/main/java/org/eclipse/ditto/connectivity/service/messaging/tunnel/TunnelChannelListener.java:61]: I'm not sure if we should "blame" the specific server in the comment, as it seems so be a problem with the client not waiting for the adjust message and an ending in a deadlock. The behavior sending a zero window size seems to be common across different SSH server implementations.

**Review by dguggemos — APPROVED** [review]

File: connectivity/service/src/main/java/org/eclipse/ditto/connectivity/service/messaging/tunnel/SshTunnelActor.java (hunk 1 above)
Comment (stmaute) [reply connectivity/service/src/main/java/org/eclipse/ditto/connectivity/service/messaging/tunnel/SshTunnelActor.java:181]: Good idea. I will change the name.

File: connectivity/service/src/main/java/org/eclipse/ditto/connectivity/service/messaging/tunnel/TunnelChannelListener.java (hunk 2 above)
Comment (stmaute) [reply connectivity/service/src/main/java/org/eclipse/ditto/connectivity/service/messaging/tunnel/TunnelChannelListener.java]: I have adjusted the comment for the workaround.

**Pull request merged**